packages/logs/README.md

```diff
@@ -252,9 +252,7 @@ The placeholders in the examples above are described below:
 
 If your Browser logs contain sensitive information that needs redacting, configure the Browser SDK to scrub sensitive sequences by using the `beforeSend` callback when you initialize the Browser Log Collector.
 
-The `beforeSend` callback function gives you access to each event collected by the Browser SDK before it is sent to Datadog, and lets you update commonly redacted properties.
-
-For examples of `beforeSend`, see [Enrich and control browser RUM data with beforeSend][5].
+The `beforeSend` callback function gives you access to each log collected by the Browser SDK before it is sent to Datadog, and lets you update commonly redacted properties.
```

Comment:
I would say "... and let's you update any property".
"commonly redacted properties" is a bit ambiguous to me.